saraho1123/whats-cookin
# Html and css basic setup

## What is the change? 
Created CSS and HTML setups.

## What does it fix?
N/A

## Is this a fix or a feature?  
Feature

## Where should the reviewer start? 
index.html 
line 1

styles.css
line 1

## How should this be tested?
If the index.html is open it should have a Whats cooking page with all necessary elements (buttons, filter & search boxes, pantry/recipe/all recipes pages.
